language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/SelectClauseNodeContext.java

```diff
@@ -73,7 +72,7 @@ public List<LSCompletionItem> getCompletions(BallerinaCompletionContext context,
                         .filter(symbol -> symbol.getName().isPresent() && !symbol.getName().get().contains("$"))
                         .filter(symbol -> completionItems
                                 .addAll(populateBallerinaFunctionCompletionItems(symbol, context)))
-                        .collect(Collectors.toList());
+                        .toList();
```

Comment:
As far as I can see we are accumilating to a list (and ignore the result) just to execute the stream here. Since we are refactoring this anyway lets convert the last "filter" to a foreach

Reply:
Fixed in [2778660](https://github.com/ballerina-platform/ballerina-lang/pull/42904/commits/277866060d74b08115089625bb7d582c095697a1)